thinkhazard/static/less/report.less

```diff
@@ -223,9 +222,45 @@ body {
   border-top-width: 1px;
   border-top-color: #e6e6e6;
   font-size: 14px;
-
   h5 {
     text-transform: uppercase;
     margin-bottom: 4px;
   }
 }
+
+#further-resources {
+  background-color: #f5f5f5;
+  margin: 50px 0px;
+  padding: 27px 30px;
+  p {
+    margin-top: 15px;
+    margin-bottom: 0px;
+    line-height: 27px;
+  }
+  ul {
+    li {
+      margin-top: 15px;
+      margin-left: 10px;
+    }
+    a {
+      color: #035e8b;
+    }
+    a:hover {
+      text-decoration: underline;
+    }
```

Comment:
_color_ and _text-decoration_ for links are to be set the whole application IMO.